sat/cli/bootsys/cabinet_power.py

```diff
@@ -202,6 +293,13 @@ def do_cabinets_power_on(args):
                      f'within expected window after being resumed.')
         raise SystemExit(1)
 
+    LOGGER.info('Recreating cronjobs which have failed to have been scheduled on time.')
```

Comment:
I think we want to put this after the hms-discovery cronjob is resumed and before we wait on it to be scheduled. Otherwise, that waiting will time out when the hms-discovery cronjob needs to be re-created.

And thanks to the way you've structured this code, it should also enable us to pull the new cronjob functions into their own `cronjob` module and then perhaps call `recreate_namespaced_stuck_cronjobs` here in the cabinet-power stage and also call it at the end of the platform-services stage. That would be a little more consistent with the ordered procedure in the docs today, which has the admin check the cronjobs during [Power On and Start the Kubernetes Management Cluster](https://github.com/Cray-HPE/docs-csm/blob/main/operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md#check-cronjobs) procedure and then again specifically for hms-discovery in step 4 of the [Power On Compute Cabinets](https://github.com/Cray-HPE/docs-csm/blob/main/operations/power_management/Power_On_Compute_Cabinets.md#power-on-cray-ex-liquid-cooled-cabinet-circuit-breakers) procedure.